app/presenters/call_for_evidence_presenter.rb

```diff
@@ -3,16 +3,30 @@ def opening_date
     display_date_and_time(content_item.opening_date_time)
   end
 
-  def opening_date_midnight?
-    Time.zone.parse(content_item.opening_date_time).strftime("%l:%M%P") == "12:00am"
-  end
-
   def closing_date
     display_date_and_time(content_item.closing_date_time, rollback_midnight: true)
   end
 
+  def on_or_at
+    opening_date_midnight? ? I18n.t("formats.call_for_evidence.on") : I18n.t("formats.call_for_evidence.at")
+  end
+
+  def opens_close_or_ran
```

Comment:
Nitpick: perhaps should be `opens_closes_or_ran` ?

Reply:
Hmm, this is still showing the old names for me. Did it get squashed into that commit correctly?

Reply:
Ah sorry, I initially misread your suggestion! It's now updated in https://github.com/alphagov/frontend/pull/4855/commits/837ce9d9f7aef1d0ea159db5638ad42a32c83a5a and subsequent commits.